xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAEResampleFFMPEG.cpp

```diff
@@ -74,6 +75,12 @@ bool CActiveAEResampleFFMPEG::Init(uint64_t dst_chan_layout, int dst_channels, i
     return false;
   }
 
+  double mix_lfe = (float) CServiceBroker::GetSettings().GetInt("audiooutput.mixlfe") / (float) 100;
+  if (mix_lfe)
+  {
+    av_opt_set_double(m_pContext, "lfe_mix_level", mix_lfe, 0);
+  }
+
```

Comment:
A lot of work has been done to remove Settings handling out of the production code an encapsulate it at one central position, so that AE can live without settings. This here is a step backward.

If we want to support this feature it should be handled the same as the other AE settings and then passed into the Resampler's Initi method.